server/sonar-search/src/main/java/org/sonar/search/EsSettings.java

```diff
@@ -116,16 +107,17 @@ private void configureFileSystem(ImmutableSettings.Builder builder) {
     builder.put("path.logs", logDir.getAbsolutePath());
   }
 
-  private void configureNetwork(ImmutableSettings.Builder builder) {
+  private void configureNetwork(Settings.Builder builder) {
     // the following properties can't be null as default values are defined by app process
-    String host = props.nonNullValue(ProcessProperties.SEARCH_HOST);
+    String host = "127.0.0.1";// props.nonNullValue(ProcessProperties.SEARCH_HOST);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this IP "127.0.0.1" address configurable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1313)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)